ecomix.py

```diff
@@ -90,19 +90,22 @@ def get_co2_from_signal_cache(start: datetime, end: datetime, country_code):
         return mean(co2_per_kw)
 
     @staticmethod
-    def get_country(latitude, longitude):
+    def get_country(latitude, longitude, country_code_default):
         try:
             location = reverse_geocode.search([(latitude, longitude)])[0]
             country_code = location["country_code"]
             return country_code
         except (UnicodeDecodeError, IndexError):
             logger.error("Can't find country for %s %s", latitude, longitude)
-            return None
+            # return None
```

Comment:
Did you had an issue finding country with your position ? 
A default country is a good alternative but I don't know why reverse geocode may trow an error...

Reply:
Hello
Because I don't have last_position field due to #50

Reply:
ok I understand, for me I have still the position field (the value is the same for one month)

Reply:
yeah but if u change privacy value, you should loose it. I experienced the same.